fhservices.cfg.j2

```diff
@@ -121,6 +121,7 @@ define service {
 
 define service {
        service_description Storage::Pod::Disk Usage
+       # REVIEW: is this being applied to every pod/container? How does it work?
```

Comment:
Thanks to @mikenairn I got a clear picture. I was missing the call to `oc exec` on all pods...

https://github.com/feedhenry/nagios-container/blob/1bc02dbd22db2741bb726944d95b43f9e796258b/plugins/default/lib/disk_usage.py#L96

The problem then is that that call doesn't handle pods with multiple containers.